pkg/runtime/pubsub/outbox.go

```diff
@@ -178,6 +178,8 @@ func (o *outboxImpl) PublishInternal(ctx context.Context, stateStore string, ope
 
 				if proj.ContentType != nil {
 					contentType = *proj.ContentType
+				} else if ct, ok := sr.Metadata[metadata.ContentType]; ok {
```

Comment:
I think this was meant to be `proj.Metadata[metadata.ContentType]`

Reply:
Yeah. sorry, to switch the PR to `release-1.14` I've mistaken to type. Fix now thanks